### PR TITLE
[README] Dark mode support: Add light versions of logos #2991

### PR DESCRIPTION
**Pull request by mihir2226**
**Description**

Add dark mode images in the README.md file.

This issue fixes https://github.com/layer5io/layer5/issues/2991

**Notes for Reviewers**
I also change the code of those images whose dark mode images are not available. So we can add those images in the future with ease.

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 14a59bcc63e9c8864c6935575cafbff601e2ab9a at: https://62e6865fa868165cbd120b3c--layer5.netlify.app

**Comment by mihir2226** [comment]
I added all the images in dark mode and all the images are changing nicely

**Review by Chadha93 — COMMENTED** [review]
@mihir2226 Please sign off your first commit to pass DCO.

**Comment by l5io** [comment]
🚀 Preview for commit c263b3c5184574550b8786379da61f13128b26a3 at: https://62e7cfb7767c754b405462f3--layer5.netlify.app

**Comment by mihir2226** [comment]
I signed off on it by deleting the file. it's not a needed file.

**Comment by mihir2226** [comment]
Can I closs my pull request. 

(Pardon me sir if I done any mistake but this is my first open source contribution and I don't know much about handling requests)

**Comment by mihir2226** [comment]
> @mihir2226 Please sign off your first commit to pass DCO.

I made some changes please try now.

**Comment by Chadha93** [comment]
@mihir2226 No worries, Let's get you covered on the DCO. :)

> hi @Chadha93, Can I close my pull request?
> 
> (Pardon me sir if I done any mistake but this is my first open source contribution and I don't know much about handling requests)

@mihir2226 No worries, Let's get you covered on the DCO. :) You have done good work on this issue.

**Comment by Chadha93** [comment]
This issue fixes #2991

**Comment by l5io** [comment]
🚀 Preview for commit 67d75fbe0327041f7e5f0158e54d43b65ce3b71a at: https://62e7f7ff15b13076208931c2--layer5.netlify.app

File: README.md [hunk 1]
```diff
@@ -98,9 +122,9 @@ alt="Layer5" align="left" /></a>
 
 <p style="clear:both;">
 <h2><a href="https://layer5.io/smi">Service Mesh Interface Conformance </a></h2>
-<a href="https://layer5.io/smi"><img src=".github/assets/images/smi-conformance/checklist.svg"
+<a href="https://layer5.io/smi"><img src=".github/assets/images/smi-conformance/checklist.svg#"
```
[the remark this answers is not in the document]
Comment (mihir2226) [reply README.md:125]: I will fix it right now

File: README.md [hunk 2]
```diff
@@ -22,9 +26,13 @@
 
 <p style="clear:both;">
 <h2><a href="https://layer5.io/meshery">Meshery</a></h2>
-<a href="https://meshery.io"><img src=".github/assets/images/meshery/meshery-logo.svg"
+<a href="https://meshery.io">
+<picture>
+  <source media="(prefers-color-scheme: dark)" srcset="https://user-images.githubusercontent.com/99991521/182012415-ccdf486f-49a0-4021-8597-7b12c9d205e5.svg#gh-dark-mode-only">
```
[the remark this answers is not in the document]
Comment (mihir2226) [reply README.md:31]: Srcset defines the set of images we will allow the browser to choose between. Using the srcset attribute has made responsive image sizing much simpler.

File: README.md [hunk 3]
```diff
@@ -1,5 +1,9 @@
 <p style="text-align:center;" align="center">
-  <img align="center" src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/layer5/layer5-no-trim.svg" width="45%" /></p>
+ <picture>
+  <source media="(prefers-color-scheme: dark)" srcset="https://user-images.githubusercontent.com/99991521/182012170-638f0731-3a83-4795-a6de-71a3614d4c7d.svg#gh-dark-mode-only">
```
[the remark this answers is not in the document]
Comment (mihir2226) [reply README.md:3]: Do you want me to change srcset to src, then I will do it right now.

**Comment by leecalcote** [comment]
Execute these commands to sign-off on your commit - https://github.com/layer5io/layer5/pull/2992/checks?check_run_id=7620256011

**Comment by mihir2226** [comment]
> Execute these commands to sign off on your commit - https://github.com/layer5io/layer5/pull/2992/checks?check_run_id=7620256011

sir, I tried to sign off, but first of all, I don't know which commit I had to sign off. because I did them in past.

**Comment by mihir2226** [comment]
@leecalcote can you give me little clear example.

**Comment by mihir2226** [comment]
did this mean my work was done?

**Pull request merged**